desktop/src/app.rs

```diff
@@ -304,38 +308,40 @@ impl App {
                                 _ => (),
                             }
 
-                            if let Some(key) = input.virtual_keycode {
-                                let key_code = winit_to_ruffle_key_code(key);
-                                let key_char = winit_key_to_char(key, modifiers.shift());
-                                match input.state {
-                                    ElementState::Pressed => {
-                                        self.player.handle_event(PlayerEvent::KeyDown {
-                                            key_code,
-                                            key_char,
+                            let key_code = winit_to_ruffle_key_code(&event);
+                            let key_char = event.text.clone().and_then(|text| text.chars().last());
```

Comment:
Hm, what about the other chars that are not `last`? :thinking:

Reply:
I think `last()` emulates the behaviour we'd have seen before.

Long term I don't know if this is something we should keep, but for now I think this is essentially the same as current

Reply:
Maybe TODO comment it then? However you feel like...

Reply:
Added!